StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -90,7 +90,12 @@ public void HandleTypeDeclaration(SyntaxNodeAnalysisContext context)
                     return;
                 }
 
-                BaseTypeDeclarationSyntax declaration = (BaseTypeDeclarationSyntax)context.Node;
+                BaseTypeDeclarationSyntax declaration = context.Node as BaseTypeDeclarationSyntax;
```

Comment:
I don't think this is the cause of the exception. This cast *should* be safe AFAIK.